lib/jenkins_pipeline_builder/extensions/publishers.rb

```diff
@@ -108,6 +108,21 @@
       end
     end
   end
+
+  version '2.0.0' do
```

Comment:
It looks like there are some options that were valid in the previous version that you're missing. I think by playing with UI you should be able to get it to write those additional attributes